samples/modelbuilder/ObjectDetection_StopSigns/StopSignDetectionML.ConsoleApp/StopSignDetectionML.ConsoleApp.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.ML" Version="1.5.0" />
+    <PackageReference Include="Microsoft.ML.ImageAnalytics" Version="1.5.0" />
+    <PackageReference Include="Microsoft.ML.OnnxTransformer" Version="1.5.0" />
+    <PackageReference Include="Microsoft.ML.OnnxRuntime" Version="1.3.0" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\StopSignDetectionML.Model\StopSignDetectionML.Model.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectCapability Include="ModelBuilderGenerated" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <None Update="test-image1.jpeg">
```

Comment:
Is the JPEG intentionally empty?

Reply:
Not sure about that. I added a jpeg as a test image. Is it ok since it's just 1 image if it's part of repo?

Reply:
Should be fine, but for some reason github was showing it as a zero-byte file in the diff.

Reply:
Nevermind, if I click "view" I can see it.

Reply:
I saw that too, but if you actually click on view the file it shows the actual picture. Seems github is just displaying that wrong.